integreat_cms/cms/migrations/0033_event_datetime.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 3.2.14 on 2022-08-21 13:03
+
+import datetime
+from django.db import migrations, models
+
+# pylint: disable=unused-argument
+def start_and_end_init(apps, schema_editor):
+    """
+    Initalize the new event datetime fields 'start' and 'end' from the respective
+    existing fields *_date and *_time.
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    events = apps.get_model("cms", "Event")
+    for event in events.objects.all():
+        event.start = datetime.datetime.combine(event.start_date, event.start_time)
+        event.end = datetime.datetime.combine(event.end_date, event.end_time)
```

Comment:
Timezones overall seem to be huge pain in the neck. I think your handling makes a lot of sense though. I think it might get a bit more complicated when we want to convert the timezones to the local times of the end-user when displaying them. I think this will need a lot of special treatment, especially when checking for all day events. We will have to be very careful with the logic and ordering of the conversions then, in order for it to function in the excepted way.